src/App.tsx

```diff
@@ -1,65 +1,41 @@
-import { Suspense } from "react";
-import { HashRouter as Router, Route, Switch } from "react-router-dom";
-
-import { Web3Provider } from "@ethersproject/providers";
-import { Web3ReactProvider } from "@web3-react/core";
+import React from "react";
+import { BrowserRouter as Router, Route, Switch } from "react-router-dom";
 
 import BigNumber from "bignumber.js";
-import { ThemeProvider, ThemeType } from "styled-components/macro";
 
-import { useAppSelector } from "./app/hooks";
-import Page from "./components/Page/Page";
-import PageLoader from "./components/PageLoader/PageLoader";
-import Whitepaper from "./components/Whitepaper/Whitepaper";
-import LastLookProvider from "./contexts/lastLook/LastLook";
-import { selectTheme } from "./features/userSettings/userSettingsSlice";
-import useSystemTheme from "./hooks/useSystemTheme";
+import HelmetContainer from "./components/HelmetContainer/HelmetContainer";
 import "./i18n/i18n";
-import GlobalStyle from "./style/GlobalStyle";
-import { darkTheme, lightTheme } from "./style/themes";
-
-let cachedLibrary: Record<string, Web3Provider> = {};
+import Home from "./pages/Home/Home";
+import Whitepaper from "./pages/Whitepaper/Whitepaper";
+import { AppRoutes } from "./routes";
 
-function getLibrary(provider: any): Web3Provider {
-  if (!cachedLibrary[provider.chainId]) {
-    cachedLibrary[provider.chainId] = new Web3Provider(provider);
-    cachedLibrary[provider.chainId].pollingInterval = 12000;
-  }
-  return cachedLibrary[provider.chainId];
-}
-//1e+9 is the highest possible number
+// 1e+9 is the highest possible number
 BigNumber.config({ EXPONENTIAL_AT: 1e9 });
 
 const App = (): JSX.Element => {
-  const theme = useAppSelector(selectTheme);
-  const systemTheme = useSystemTheme();
-  const renderedTheme: ThemeType =
-    theme === "system" ? systemTheme : (theme as ThemeType);
-
   return (
-    <Router>
-      <Switch>
-        <Route exact path="/whitepaper">
-          <Whitepaper />
-        </Route>
-
-        <Route path="/:tokenFrom?/:tokenTo?">
-          <ThemeProvider
-            theme={renderedTheme === "dark" ? darkTheme : lightTheme}
-          >
-            <Web3ReactProvider getLibrary={getLibrary}>
-              {/* Suspense needed here for loading i18n resources */}
-              <Suspense fallback={<PageLoader />}>
-                <LastLookProvider>
-                  <Page />
-                </LastLookProvider>
-              </Suspense>
-            </Web3ReactProvider>
-            <GlobalStyle />
-          </ThemeProvider>
-        </Route>
-      </Switch>
-    </Router>
+    <>
+      <HelmetContainer
+        title={"AirSwap - Open source web app for instant swaps"}
+        description={"AirSwap powers peer-to-peer trading."}
```

Comment:
Adding Ethereum here would help with SEO. Also gas free instead of instant? I don't know if instant brings any value, seeing as every other swap service are just as fast.
Furthermore, what do you think of localization? Probably a bit of a hassle if it doesn't work out of the box with the locale function